### ISSUE TITLE
Added glossary entry for Assisted Installer

### ISSUE THREAD

**Issue by rolfedh** [redhat-documentation/vale-at-red-hat#423]
New guideline [added in to SSG in January](https://github.com/redhat-documentation/supplementary-style-guide/blob/main/HISTORY.md#2023-january):
- Added glossary entry for [Assisted Installer](https://redhat-documentation.github.io/supplementary-style-guide/#assisted-installer)

**Comment by aireilly** [comment]
partially addressed in https://github.com/redhat-documentation/vale-at-red-hat/pull/409

Needs moving to the CaseSenstiveTerms.yml IMO. `AI|assisted installer: Assisted Installer`

**Comment by michellemacrh** [comment]
Hey @aireilly 👋 is this still an active issue? Just to clarify the ask: move `AI|assisted installer: Assisted Installer` entry to CaseSensitiveTerms.yml

**Comment by aireilly** [comment]
Yes, however, I think we need to change it to: `assisted installer: Assisted Installer` in CaseSensitiveTerms.yml and remove it from TermsErrors.yml 

AI is commonly used to refer to Artificial Intelligence, so I think it would be best to remove it altogether.
